cmd/palomar/README.md

```diff
@@ -44,8 +40,7 @@ index up to date you will periodcally need to scrape the data.
 ## API
 
 ### `/index/:did`
-Index the content in the given users repo. Keeps track of the last repo update
-and only fetches incremental changes
+Indexes the content in the given user's repository. It keeps track of the last repository update and only fetches incremental changes.
 
 ### `/search?q=QUERY`
-Very simple case-insensitive search results from across the entire app.
+Performs a simple, case-insensitive search across the entire application.
```

Comment:
these routes are actually different now, probably needs some updating

Reply:
I've tried to determine what these are, and I can't sort it out. Over time I'll have a better understanding of how everything interacts, but it isn't clear to me yet.